src/App.js

```diff
@@ -1,9 +1,84 @@
 import React from 'react';
 import './App.css';
+import Button from "./components/Button";
+import Product from "./components/Product";
+import Tile from "./components/Tile";
+
+// afbeeldingen
+import bagOne from "./assets/bag_1.png"
+import bagTwo from "./assets/bag_2.png"
+import bagThree from "./assets/bag_3.png"
+import bagFour from "./assets/bag_4.png"
+import brand from "./assets/brand.png"
+import ourStory from "./assets/our_story.png"
+
 
 function App() {
   return (
-      <h1>Handbags & Purses</h1>
+      <>
+        <h1>Handbags & Purses</h1>
+        <nav>
+            <Button
+                text="to the collection"
```

Comment:
@TinekeV 
Bij het button-component in de APP zou je de clickhandler kunnen toepassen ipv in de Button.js, conform onderstaand voorbeeld:
`        <Button isDisabled={false} clickHandler={() => console.log("Shop all bags!")}>
          Shop all bags
        </Button>`